packages/gatsby-cli/src/index.ts

```diff
@@ -36,7 +34,7 @@ process.on(`unhandledRejection`, reason => {
   // across versions and crash
 
   // reason can be anything, it can be a message, an object, ANYTHING!
-  // we convert it to an error object so we don't crash on structured error validation
+  // we convtert it o an error object so we don't crash on structured error validation
```

Comment:
this changes introduced 2 typos?
```suggestion
  // we convert it to an error object so we don't crash on structured error validation
```